sirepo/template/webcon.py

```diff
@@ -356,6 +356,205 @@ def get_settings_report(run_dir, data):
 #    raise RuntimeError('{}: unknown simulation frame model'.format(data['modelName']))
 
 
+# use jinja?
+def jupyter_notebook_for_model(data, model):
+    # notebooks are JSON files
+    from sirepo.template.template_common import JupyterNotebook
+    nb = JupyterNotebook(SIM_TYPE, data)
+    nb.add_imports(
+        {
+            'numpy': []
+        }
+    )
+
+    nb.add_markdown_cell(['## Function definitions'])
+    nb.add_code_cell([
+        template_common.render_jinja(SIM_TYPE, {}, name='common.py')
+    ])
+    nb.add_code_cell([
+        template_common.render_jinja(SIM_TYPE, {}, name='analysis.py')
```

Comment:
This does not appear to be jinja. Why would it be jinja if there is no environment `{}`?